packages/typespec-vscode/src/extension.ts

```diff
@@ -297,9 +297,9 @@ async function resolveTypeSpecServer(context: ExtensionContext): Promise<Executa
       const command =
         process.platform === "win32" && !serverPath.endsWith(".cmd")
           ? `${serverPath}.cmd`
-          : "tsp-server";
+          : serverPath;
```

Comment:
@timotheeguerin , the original code looks like a bug to me. i.e. when .../tsp-server.cmd is given, the original code will change it to "tsp-server" which won't work in windows. so update it to serverPath. Please help to confirm. thanks.

Reply:
yeah seems like a bug